### PR TITLE
Adding  autodocumentation action

### PR DESCRIPTION
**Pull request by Poetickz**
# Description

* Adds auto-doc action & removes `_build` directory from `.gitignore`

Fixes #TEPYG-75

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Running `pytest`

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/autodoc.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - name: Building auto documentation 
+        uses: ammaraskar/sphinx-action@master
+        with: 
+          docs-folder: "docs/"
+      
+      - name: Commit changes
+        run: |
+          git clone https://github.com/Poetickz/color-cone-algorithm.git
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
```
Comment (EBKH) [inline .github/workflows/autodoc.yml:33]: ocupan ser este user? Pensaba en usar a valbot :P

Comment (Poetickz) [reply .github/workflows/autodoc.yml:33]: Puede ser cualquier nombre >-<

Comment (EBKH) [reply .github/workflows/autodoc.yml:33]: This guy https://github.com/valiotbot :P

Comment (Poetickz) [reply .github/workflows/autodoc.yml]: Done.

**Review by EBKH — APPROVED** [review]

**Pull request merged**